lib/ci-helper.ts

```diff
@@ -62,6 +64,54 @@ export class CIHelper {
         this.urlRepo = `${this.urlBase}${this.config.repo.name}/`;
     }
 
+    public async setupGitHubAction(): Promise<void> {
+        // help dugite realize where `git` is...
```

Comment:
Would the `find-git-exec` package be more environment agnostic to get the git location?